rosie/federal_senate/tests/test_adapter.py

```diff
@@ -0,0 +1,41 @@
+import shutil
+import os
+from tempfile import mkdtemp
+from unittest import TestCase
+from unittest.mock import patch
+from shutil import copy2
+
+import pandas as pd
+
+from rosie.federal_senate.adapter import Adapter as subject_class
+from rosie.federal_senate.adapter import COLUMNS as ADAPTER_COLUMNS
+
+class TestAdapter(TestCase):
+    @classmethod
```

Comment:
Why do we need this decorator here and `setUpClass`? I have the impression that an undecorated `setUp` method would work, but surely I might be wrong…